### ISSUE TITLE
Accessibility: Properties tearsheet does not have associated name

### ISSUE THREAD

**Issue by nmgokhale** [elyra-ai/canvas#1391]
Ref: 3484

Fix following accessibility violations in Tearsheet panel -

- Interactive component with ARIA role 'dialog' does not have a programmatically associated name
- Interactive component with ARIA role 'tabpanel' does not have a programmatically associated name
<img width="761" alt="dialog-name-violation" src="https://user-images.githubusercontent.com/25124000/223822600-77e32e04-9012-4cfd-bb6c-9e0f3e7b6419.png">
<img width="757" alt="tabpanel-violation" src="https://user-images.githubusercontent.com/25124000/223822609-75cc5819-d5a9-4779-b1dc-e1073c7043b6.png">

![Screenshot 2023-03-08 at 11 20 57 AM](https://user-images.githubusercontent.com/25124000/223822231-f7e881e0-4997-495a-b81a-6119b0373083.png)
